addons/analytic/models/analytic_mixin.py

```diff
@@ -112,3 +112,7 @@ def _apply_analytic_distribution_domain(self, domain):
             ('analytic_distribution_search', leaf[1], leaf[2]) if len(leaf) == 3 and leaf[0] == 'analytic_distribution' and isinstance(leaf[2], str) else leaf
             for leaf in domain
         ]
+
+    def _get_analytic_accounts(self):
```

Comment:
I will say that what you're getting are ids, so the name of the function should reflect it.

And I actually don't know if this should be private or public. Couldn't we find a use for it in js for example?